kdl_parser_py/package.xml

```diff
@@ -24,11 +24,12 @@
 
   <build_depend version_gte="1.3.0">orocos_kdl</build_depend>
   <build_depend>urdf</build_depend>
-  <build_depend>rostest</build_depend>
 
-  <run_depend version_gte="1.3.0">orocos_kdl</run_depend>
-  <run_depend>urdf</run_depend>
-  <run_depend>urdfdom_py</run_depend>
-  <run_depend>python_orocos_kdl</run_depend>
+  <exec_depend version_gte="1.3.0">orocos_kdl</exec_depend>
+  <exec_depend>urdf</exec_depend>
```

Comment:
same here this dependency should be exported

Reply:
Added `<build_export_depend>` for the tags that were converted from `<run_depend>`, but to be honest I'm not sure what that means for a python dependency. When does a python dependency need to be `build_export_depend`ed?